templates/index.html

```diff
@@ -0,0 +1,98 @@
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
+            <a href="#">My Logo</a>
+          </div>
+
+          <div class="nav__container__parnav__bars">
+            <div class="nav__container__parnav__bars__responsive">
+              <img
+                src="..//static/imgs/Union@2x.png"
+                alt="Union@2x"
+                class="responsive nav__container__parnav__bars__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <!-- end nav -->
+    <!-- start sectionone -->
+    <section class="sectionone" id="sectionone">
+      <div class="overlay">
+        <div class="sectionone__heroimage">
+          <h2>
+            I’m Alexandra <br />
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through <br />
+            some of my work and experience! If you like what you see and have a
+            project you need <br />
+            coded, don’t hestiate to contact me.
+          </p>
+          <span>LET’S CONNECT</span>
+          <div class="social">
+            <div class="social__responsive">
+              <img
+                src="../static/imgs/twitter.png"
+                alt="twitter"
+                class="social__responsive__respo"
+              />
+            </div>
+            <div class="social__responsive">
+              <img
+                src="../static/imgs/Linkedin icon.png"
+                alt="Linkedin"
+                class="social__responsive__respo"
+              />
+            </div>
+            <div class="social__responsive">
+              <img
+                src="../static/imgs/medium.png"
+                alt="medium"
+                class="social__responsive__respo"
+              />
+            </div>
+            <div class="social__responsive">
+              <img
+                src="../static/imgs/github.png"
+                alt="github"
+                class="social__responsive__respo"
+              />
+            </div>
+            <div class="social__responsive">
+              <img
+                src="../static/imgs/angellist.png"
+                alt="angellist"
+                class="social__responsive__respo"
+              />
+            </div>
+          </div>
```

Comment:
Kindly note that this list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag) 🤗
- _Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href._